build.xml

```diff
@@ -1444,75 +1447,6 @@
     </sequential>
   </macrodef>
 
-  <!-- Will not generate a junit report or fail on error since it is called in parallel for test-compression
-       That is taken care of by testparallel -->
-  <macrodef name="testlist-compression">
-    <attribute name="test.file.list" />
-    <attribute name="testlist.offset" />
-    <sequential>
-      <property name="compressed_yaml" value="${build.test.dir}/cassandra.compressed.yaml"/>
-      <concat destfile="${compressed_yaml}">
-          <fileset file="${test.conf}/cassandra.yaml"/>
-          <fileset file="${test.conf}/commitlog_compression.yaml"/>
-      </concat>
-      <testmacrohelper inputdir="${test.unit.src}" filelist="@{test.file.list}" poffset="@{testlist.offset}"
-                       exclude="**/*.java" timeout="${test.timeout}" testtag="compression">
-        <jvmarg value="-Dlegacy-sstable-root=${test.data}/legacy-sstables"/>
-        <jvmarg value="-Dinvalid-legacy-sstable-root=${test.data}/invalid-legacy-sstables"/>
-        <jvmarg value="-Dcassandra.test.compression=true"/>
-        <jvmarg value="-Dcassandra.ring_delay_ms=1000"/>
-        <jvmarg value="-Dcassandra.tolerate_sstable_size=true"/>
-        <jvmarg value="-Dcassandra.config=file:///${compressed_yaml}"/>
-        <jvmarg value="-Dcassandra.skip_sync=true" />
-        <jvmarg value="-Dcassandra.config.loader=org.apache.cassandra.OffsetAwareConfigurationLoader"/>
-      </testmacrohelper>
-    </sequential>
-  </macrodef>
-
-  <macrodef name="testlist-cdc">
-    <attribute name="test.file.list" />
-    <attribute name="testlist.offset" />
-    <sequential>
-      <property name="cdc_yaml" value="${build.test.dir}/cassandra.cdc.yaml"/>
-      <concat destfile="${cdc_yaml}">
-        <fileset file="${test.conf}/cassandra.yaml"/>
-        <fileset file="${test.conf}/cdc.yaml"/>
```

Comment:
is there any way of maintaining this feature?  it is cumbersome to keep all the different test cassandra.yaml's up to date…